### ISSUE TITLE
Caps Lock held down is not passed through to the VM

### ISSUE THREAD

**Issue by UTMDiscordBot** [utmapp/UTM#4330]
Okay so holding capslock while pressing other keys is an extremely common practice in screenreaders, but this does not work with UTM in a VM. Why<br><br><h2>Created using UTMbot</h2>Questioner: neverseen#0001<br>Moderator: BluDood#0001

**Comment by UTMDiscordBot** [comment]
You might want to add the capslock passthrough problem is macOS only, it works just fine on iOS.<br><br><h2>Added comment using UTMbot</h2>Questioner: neverseen#0001<br>Moderator: BluDood#0001

**Comment by osy** [comment]
I think the issue is with the new way we handle caps lock introduced some updates ago. Will have to investigate it more.

**Comment by osy** [comment]
Seems like the global preference is already implemented to workaround this.
